Peer.GitHub/GraphQL/PullRequestSearch/PullRequest.cs

```diff
@@ -41,10 +48,13 @@ public Domain.PullRequest Into()
             var totalComments = ReviewThreads.Nodes.Count;
             var activeComments = ReviewThreads.Nodes.Count(t => !t.IsResolved);
             var suites = Commits.Nodes.SelectMany(x => x.Commit.CheckSuites.Nodes);
-            var checks = suites.SelectMany(suite => suite.CheckRuns.Nodes.Where(run => run.Url != null).Select(z => z.Into())).ToList();
+            var checks = suites.SelectMany(suite => suite.CheckRuns.Nodes.Where(run => run.Url != null)
+                .Select(z => z.Into()))
```

Comment:
z? What are you going to do next charge me $11 to watch you smoke a vegan bicycle?

Reply:
(I know it's existing)